{"uid":"d226d7018243506d","name":"Archive a card","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_archive_card","historyId":"4151bed777d70c35651f4626d00b86b5","time":{"start":1735150876497,"stop":1735150908985,"duration":32488},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f9ca719e7e0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a0efa87953c27f9d0198641d101d1829\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"a0efa87953c27f9d0198641d101d1829\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f9ca716e2a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f9ca719e7e0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a0efa87953c27f9d0198641d101d1829\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC006\n    @allure.title('Archive a card')\n    def test_archive_card(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:188: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1735150876232,"stop":1735150876496,"duration":264},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1735150876496,"stop":1735150876497,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f9ca719e7e0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a0efa87953c27f9d0198641d101d1829\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"a0efa87953c27f9d0198641d101d1829\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f9ca716e2a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f9ca719e7e0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a0efa87953c27f9d0198641d101d1829\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC006\n    @allure.title('Archive a card')\n    def test_archive_card(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:188: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","steps":[{"name":"Log in to Trello account","time":{"start":1735150881497,"stop":1735150908984,"duration":27487},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 188, in test_archive_card\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1735150881497,"stop":1735150884768,"duration":3271},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1735150884768,"stop":1735150908982,"duration":24214},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"5bff77edbdff9b2","name":"log","source":"5bff77edbdff9b2.txt","type":"text/plain","size":63296}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1735150909139,"stop":1735150909212,"duration":73},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC006"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1372-862"},{"name":"thread","value":"3622-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":10,"skipped":0,"passed":32,"unknown":0,"total":44},"items":[{"uid":"d17e727f3a76c1e4","status":"failed","statusDetails":"AssertionError: assert 1 == 0","time":{"start":1735088748609,"stop":1735088780384,"duration":31775}},{"uid":"7ea099cce60b2466","status":"passed","time":{"start":1735064419152,"stop":1735064450963,"duration":31811}},{"uid":"825b50257751efc3","status":"passed","time":{"start":1735002342502,"stop":1735002374320,"duration":31818}},{"uid":"20c1854dcd491f2a","status":"passed","time":{"start":1734978026823,"stop":1734978058674,"duration":31851}},{"uid":"b57764b64ee78d48","status":"passed","time":{"start":1734916024327,"stop":1734916056159,"duration":31832}},{"uid":"3c695bff913369c7","status":"passed","time":{"start":1734891544514,"stop":1734891576590,"duration":32076}},{"uid":"6941b1948d86c65c","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1734829996568,"stop":1734830029140,"duration":32572}},{"uid":"518caa36e8aab9d8","status":"passed","time":{"start":1734805149528,"stop":1734805181329,"duration":31801}},{"uid":"84fc8400ad3195c3","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1734743087487,"stop":1734743120055,"duration":32568}},{"uid":"94206452dcca79f2","status":"passed","time":{"start":1734718838227,"stop":1734718870082,"duration":31855}},{"uid":"3133d3eacba3ff1f","status":"passed","time":{"start":1734656825416,"stop":1734656857295,"duration":31879}},{"uid":"12bb31303b9de6d9","status":"passed","time":{"start":1734632467543,"stop":1734632499513,"duration":31970}},{"uid":"2cc50933b1a675a5","status":"passed","time":{"start":1734570548407,"stop":1734570580393,"duration":31986}},{"uid":"dad79e705018163b","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1734546136967,"stop":1734546169481,"duration":32514}},{"uid":"6c1c3b1343c066f3","status":"passed","time":{"start":1734484105873,"stop":1734484137922,"duration":32049}},{"uid":"fbd0dc37598964a7","status":"passed","time":{"start":1734459780450,"stop":1734459812486,"duration":32036}},{"uid":"1b787dcbbf5fc018","status":"passed","time":{"start":1734397883717,"stop":1734397915899,"duration":32182}},{"uid":"a228cff3fe09c5b8","status":"failed","statusDetails":"AssertionError: assert 2 == 0","time":{"start":1734373380111,"stop":1734373411807,"duration":31696}},{"uid":"1f36e5dd96e68fe7","status":"passed","time":{"start":1734311674559,"stop":1734311706833,"duration":32274}},{"uid":"bdb9490770f7255e","status":"passed","time":{"start":1734286829833,"stop":1734286861877,"duration":32044}}]},"tags":["TC006","TC000"]},"source":"d226d7018243506d.json","parameterValues":[]}